### PR TITLE
Fix: calls mergeIntoState in SSR example

### PR DESCRIPTION
**Pull request by Jascha-Sundaresan**
This fixes issue #75 

Nothing was wrong w/ the library. The example just called an update function that was previously defined within the example. That function is referred to as 'mergeIntoState' in the library.

-----

The maintainers of this repo require that all pull request submitters adhere to the following:

- [x] I have read and will comply with the [contribution guidelines](https://github.com/FormidableLabs/freactal/blob/master/CONTRIBUTE.md).
 _(required)_
- [x] I have read and will comply with the [code of conduct](https://github.com/FormidableLabs/freactal/blob/master/CONTRIBUTE.md).
 _(required)_
- [x] All related documentation has been updated to reflect the changes made. _(required)_
- [x] My commit messages are cleaned up and ready to merge. _(required)_

The maintainers of this repository require you to select the semantic version type that
the changes in this pull request represent.  Please select one of the following:
- [ ]  major
- [ ]  minor
- [x]  patch
- [ ]  documentation only

### CODE REVIEW

**Comment by ryan-roemer** [comment]
@Jascha-Sundaresan -- Thanks. Looking at this now.

File: spec/.babelrc [hunk 1]
```diff
@@ -4,6 +4,6 @@
     "transform-react-jsx"
   ],
   "presets": [
-  	["latest", { "es2017": false }]
+    ["env", { "targets": { "node": "current" } }]
```
[the remark this answers is not in the document]
Comment (kenwheeler) [reply spec/.babelrc]: Is there anything we would need there to support SSR?

[the remark this answers is not in the document]
Comment (kenwheeler) [reply spec/.babelrc]: wait, probably not

[the remark this answers is not in the document]
Comment (Jascha-Sundaresan) [reply spec/.babelrc]: this is just in the spec, not the main babelrc.

The specs run on node, not the browser.

**Comment by Jascha-Sundaresan** [comment]
@ryan-roemer the babelrc you're commenting on is _not_ the main babelrc which is configured to support browsers.  The babelrc you are commenting on is in the spec folder and is only for running specs, which run on node.

File: spec/.babelrc (hunk 1 above)
[the remark this answers is not in the document]
Comment (ryan-roemer) [reply spec/.babelrc]: Ah. Gotcha!

Let's coalesce / harmonize the changes then. Can you remove `babel-preset-latest` from package.json and have root `.babelrc` switch over too? Thanks!

[the remark this answers is not in the document]
Comment (Jascha-Sundaresan) [reply spec/.babelrc]: heh.. both of those are already done. :)

Babel preset latest is no longer in the package.json and the root .babelrc is set to [env]

[the remark this answers is not in the document]
Comment (Jascha-Sundaresan) [reply spec/.babelrc]: incidentally, I'd like to add one more thing to this commit now.

I'd like to switch the peerDependency on react from "^15.5.4" to "^15.5.4 || ^16.0.0"

[the remark this answers is not in the document]
Comment (Jascha-Sundaresan) [reply spec/.babelrc]: chai-enzyme is going to need to get bumped, too. it's got a peerDependency on a deprecated lib: react-addons-test-utils

maybe we should bump the react15 ver to ^15.6.2 so people are forced to use a version of react that gives deprecation warnings to get them to react 16 in order to use freactal w/o yarn throwing a warning.

**Pull request merged**